programs/svm-spoke/src/lib.rs

```diff
@@ -194,11 +194,11 @@ pub mod svm_spoke {
             proof,
         )
     }
-    pub fn bridge_tokens_to_hub_pool(
+    pub fn bridge_tokens_to_hub_pool<'info>(
```

Comment:
do we strictly need this info lifetime here? its not being passed in `Context<BridgeTokensToHubPool>` below